code/modules/projectiles/guns/medbeam.dm

```diff
@@ -1,6 +1,6 @@
 /obj/item/weapon/gun/medbeam
 	name = "Medical Beamgun"
-	desc = "Delivers medical nanites in a focused beam."
+	desc = "Delivers medical nanites in a focused beam. On it is a label saying: 'Warning: Crossing the beam from this device with another is highly not recommended'."
```

Comment:
"Delivers medical nanites in a focused beam - crossing two or more beams is not recommended"